main.py

```diff
@@ -125,6 +141,14 @@ def _get_link_from_interface(self, interface):
                 return link
         return None
 
+    @staticmethod
+    def _load_intf_available_tags(interface, intf_dict):
```

Comment:
@ajoaoff last you last reviewed, I had to update this static method here on this commit https://github.com/kytos-ng/topology/pull/46/commits/efb70706a8e5edeb473297d6c10dea0629f35c8f, now it's only relying if the `available_tags` exists, prior to that we had margin for an issue since empty loaded tags wouldn't load (and consequently it would use the full range that kytos sets by default), which would case issues one day.